apps/daffio/src/app/core/footer/footer.module.ts

```diff
@@ -30,10 +35,14 @@ import {
     DaffioNewsletterModule
   ],
   declarations: [
-    DaffioFooterComponent
+    DaffioFullFooterComponent,
+    DaffioSimpleFooterComponent,
+    DaffioSubFooterComponent
   ],
   exports: [
-    DaffioFooterComponent
+    DaffioFullFooterComponent,
```

Comment:
This component is never used, remove it from both `declarations` and `exports`.

Reply:
Resolved in 8f671cf